Readers/interface/RecoHadTauReader.h

```diff
@@ -78,6 +82,7 @@ class RecoHadTauReader : public ReaderBase
 
   TauID tauID_;
   TauESTool * tauESTool_;
```

Comment:
Please purge the code from `TauESTool`, we don't need it anymore if we load the corrections from `correctionlib`.